backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc

```diff
@@ -13,8 +13,7 @@ OADP supports CSI snapshots on the following:
 * Red Hat OpenShift Data Foundation
 * Any other cloud storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
-:FeatureName: The OADP built-in Data Mover
-include::snippets/technology-preview.adoc[]
+include::snippets/datamover-1-3-technology-preview.adoc[]
```

Comment:
@CarmiWisemon Suggest removing the tech preview snippet and retain the following statement as an IMPORTANT admonition in the assembly instead:

OADP 1.3 Data Mover moves from being a Technology Preview feature only, to being GA from OADP 1.3.1.